test/specs/test_calc.unit.js

```diff
@@ -33,5 +33,17 @@ let Calculator = new Calc;
         it('return multiplying -2 and 2 equal to -4', () => {
             expect(Calculator.mul(-2, 2)).to.equal(-4);
         });
+        it('returns NaN if the value of one of the parameters is NaN', () => {
+            expect(isNaN(Calculator.mul(2, NaN))).to.equal(true);
+        });
+        it('returns NaN if the value of one of the parameters is a string', () => {
```

Comment:
The description is not currently right, because you I can use '1' as argument, which is string.